utils/android/README.md

```diff
@@ -0,0 +1,14 @@
+
+# Build Android Toolchain
```

Comment:
Just a suggestion: I think expanding on what "Android toolchain" means here would help people who have some interest, but don't quite understand what a "toolchain" is, to use the scripts and tools in here. Ideally, I think this README should answer the question: "why would I want to build an Android toolchain?"